examples/collector-agent-host-network-disabled/README.md

```diff
@@ -0,0 +1,9 @@
+# Example of chart configuration
+
+## Deploy the Splunk Collector in agent mode and disable using the host network
+
+- This configuration will install the collector as an agent deployment only.
+- Disabling this value will affect monitoring of some control plane components.
+  - TODO: Add metrics that would be missing
```

Comment:
Host network metrics likely will be reported incorrectly. We need to investigate that and document it before merging